bower.json

```diff
@@ -11,6 +11,8 @@
     "location",
     "position",
     "navigation",
+    "api",
```

Comment:
dont' think we need these. they're vague

Reply:
@ebidel I've removed the `permission` keyword. But IMO we should keep the `api` keyword. `<geo-location>` web component is a part of my [progressive-elements](https://www.webcomponents.org/collection/FluorescentHallucinogen/progressive-elements) collection that is a set of web components for [modern web APIs](https://whatwebcando.today) such as Payment Request API, Media Session API, etc. People can find these web components by the `api` keyword.